koku/masu/database/report_db_accessor_base.py

```diff
@@ -118,8 +154,17 @@ def _execute_trino_raw_sql_query(self, sql, *, sql_params=None, context=None, lo
         )
         return results
 
+    @handle_trino_external_error
     def _execute_trino_raw_sql_query_with_description(
-        self, sql, *, sql_params=None, context=None, log_ref="Trino query", attempts_left=0, conn_params=None
+        self,
+        sql,
+        *,
+        sql_params=None,
+        context=None,
+        log_ref="Trino query",
+        attempts_left=0,
+        conn_params=None,
+        trino_external_error_retries=3,
```

Comment:
These should be a parameters to the decorator, not this function.
```suggestion
        conn_params=None,
```